python/py-pyobjc/Portfile

```diff
@@ -2,16 +2,18 @@
 
 PortSystem          1.0
 PortGroup           python 1.0
+PortGroup           bitbucket 1.0
```

Comment:
What's wrong with the pypi release?

Reply:
The source repository contains all the frameworks for a given version of macOS — duplicating that list seems excessive. The somewhat unusual installer contains the ability to install them all in the correct order.

Reply:
Upstream publishes them all to pypi as separate packages, is there not a good reason for that? Do we want to make users install pyobjc frameworks they don't need?